salt/grains/esxi.py

```diff
@@ -104,7 +106,7 @@ def _grains():
                 port=port,
             )
             GRAINS_CACHE.update(ret)
-    except KeyError:
+    except KeyError as e:
```

Comment:
If we're not going to use `e` let's remove `as e` please.

Reply:
Done.